packages/backend/src/services/CsvService/CsvService.ts

```diff
@@ -956,15 +1118,13 @@ export class CsvService extends BaseService {
                 void archive.finalize(); // This finalize doesn't wait for the files to be written
             });
 
-        const csvFiles = await this.getCsvsForDashboard(
+        const csvFiles = await this.getCsvsForDashboard({
```

Comment:
was getting too hard to maintain this function with many args, so I changed it to a single object